test/integration/configs/config.node-4.json

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
these config files should not be tracked, these are generated automatically during the integration test run, please remove all the configs `config.node-0.json` to `config.node-9.json`